geth/jail/jail.go

```diff
@@ -201,18 +201,12 @@ func (jail *Jail) Send(call otto.FunctionCall) (response otto.Value) {
 
 	// Execute the requests.
 	for _, req := range reqs {
-		var resErr error
-		var res *otto.Object
-
-		switch req.Method {
-		case params.SendTransactionMethodName:
-			res, resErr = jail.policy.ExecuteSendTransaction(req, call)
-		default:
-			res, resErr = jail.policy.ExecuteOtherTransaction(req, call)
-		}
+		// TODO(adam): these calls should be called async and results collected
```

Comment:
@tiabc am I right here?

Reply:
I see. Hm, I think it's safer to leave it for the time being as we have started testing the upstream RPC client and new commands will require filtering.

Reply:
I'd remove this TODO and leave just a note about possible optimisation here and explaining why it's not currently done this way.